sources/models/src/modeled_types/mod.rs

```diff
@@ -45,6 +45,9 @@ pub mod error {
         #[snafu(display("Invalid Kubernetes authentication mode '{}'", input))]
         InvalidAuthenticationMode { input: String },
 
+        #[snafu(display("Invalid bootstrap container mode '{}'", input))]
```

Comment:
Nit: I think your error message in the bootstrap-containers binary will basically repeat this statement twice.

Reply:
Updated the code to show the message just once